README.md

```diff
@@ -0,0 +1,2 @@
+# Companies House Payments API Service
+The Companies House API Service for handling payments.
```

Comment:
Can you add the configs to the README please. See the example here: https://github.com/companieshouse/styleguides/blob/master/READMEs.md

Reply:
Done